--- graphviz-2.49.0/plugin/core/gvloadimage_core.c.bak	2021-09-07 12:40:50.450694232 +0900
+++ graphviz-2.49.0/plugin/core/gvloadimage_core.c	2021-09-07 12:41:01.027877866 +0900
@@ -186,7 +186,7 @@ static void core_loadimage_ps(GVJ_t * jo
 			us->data = NULL;
 #else
 		us->data = malloc(statbuf.st_size);
-		read(fd, us->data, statbuf.st_size);
+		read(fd, us->data, (size_t) statbuf.st_size);
 #endif
 		us->must_inline = TRUE;
                 break;
